roofit/roofitcore/src/RooMinimizerFcn.cxx

```diff
@@ -541,4 +149,14 @@ double RooMinimizerFcn::DoEval(const double *x) const {
   return fvalue;
 }
 
+std::string RooMinimizerFcn::getFunctionName() const
```

Comment:
These two functions `getFunctionName` and `getFunctionTitle` can also be inlined in the header to allow for better compiler optimization and improve code readability.

Reply:
Maybe, maybe not. I left some comments to bring down the includes in the headers. If you follow that route, it's actually better to forward declare `RooAbsReal`, and have those functions in the `cxx`.
If `RooAbsReal` needs to be included in the header, I would suggest inlining.

Reply:
Good point, if inlining means to add mode includes that would otherwise not be necessary it should not be done.